frontend/components/shelter/home/PetsTable/index.tsx

```diff
@@ -1,55 +1,57 @@
 import { ImportOutlined, PlusOutlined } from "@ant-design/icons";
+import { Animal } from "@contract";
 import { Button, Input, Table } from "antd";
 import { ColumnsType } from "antd/lib/table/interface";
-import { PetData, Species, Status } from "common/enums";
 import React from "react";
 import styled from "styled-components";
-import TablePill, { TablePillType } from "./components/Pill";
 import TableName from "./components/Name";
+import TablePill, { TablePillType } from "./components/Pill";
 import styles from "./PetsTable.module.css";
 
 const { tableHeader, actionButton } = styles;
 const { Search } = Input;
 
-const Container = styled.div`
-	margin: 24px 34px;
-	padding: 24px;
-	background: var(--color-white);
-`;
-
-const PetTableDisplay = () => {
+const PetTableDisplay = ({ petData }: { petData: Animal.Attributes[] }) => {
 	/* commenting out for now as not implemented yet */
 	// const [searchText, setSearchText] = useState("");
 	// const [searchedColumn, setSearchedColumn] = useState("");
+
 	const onSearch = () => {
 		console.log("search");
 	};
 	const onViewMore = (id: string) => {
 		console.log("clicked view more ", id);
 	};
 
-	const columns: ColumnsType<Omit<PetData, "acquired" | "breed">> = [
+	const columns: ColumnsType<Animal.Attributes> = [
```

Comment:
any idea why "acquired" and "breed" were omitted in the first place? I checked the type and "breed" is present on Animal.Attributes.

Reply:
i think the original schema we were using for frontend didn't have it. Basically the schema's weren't aligned from back to front and from front to column. but should be fine now